### PR TITLE
Setup for Archive PyCon TW 2023

### PR DESCRIPTION
**Pull request by rockleona**
## Types of changes

* **Breaking change**
  -  Archive PyCon TW 2023
  -  Setup as "static" mode to generate static files
  -  All `dispatch` method are changed into `async asyncData()`

### CODE REVIEW

**Comment by rockleona** [comment]
@josix , could you help me out due to some *possible* misconfiguration that can not build specific pages.

**Comment by josix** [comment]
It works well on my laptop and netlify, I think we could go on to review it. @baby230211 could you help review it, thanks!

**Comment by rockleona** [comment]
Hi @josix  and @baby230211, I found a misplaced component `venueMap` in `pages`, already moved it under `components`.

Also I re-generate the code again and it works, probably I used the wrong `Auth_Token` last time, thanks to Josix for helping me out.

**Review by josix — CHANGES_REQUESTED** [review]
Hi @rockleona seems there are some ci issues need to be fixed cloud you help fix it, thanks!

**Comment by rockleona** [comment]
Hi @josix, I already fix the linter error and seems like it's solved, please check.

File: package.json [hunk 1]
```diff
@@ -7,6 +7,7 @@
         "build": "nuxt build",
         "start": "nuxt start",
         "generate": "nuxt generate",
+        "preview": "nuxt preview",
```
Comment (baby230211) [inline package.json:10]: just curious about why adding preview command.
this might be not related to this PR.

**Comment by baby230211** [comment]
This comment is just to mention previous related [PR](https://github.com/pycontw/pycontw-frontend/pull/380) for this same feature.

File: package.json (hunk 1 above)
Comment (rockleona) [reply package.json:10]: Yes, I accidentally add this line when I try to preview in local machine.
I'll remove it later.

**Review by baby230211 — APPROVED** [review]
LGTM

**Review by josix — APPROVED** [review]

**Comment by josix** [comment]
Thanks @rockleona! BTW, could you help follow the [instruction](https://github.com/pycontw/pycon_archive_past_website) to archive the 2023 site and upload it to [pycon_archive_past_website](https://github.com/pycontw/pycon_archive_past_website).

**Pull request merged**